src/components/ui/Text.tsx

```diff
@@ -505,3 +506,13 @@ export function TrashButtonText({ children }: ChildrenProp) {
     </span>
   )
 }
+
+const largeText = classnames(
+  fontFamily('font-primary'),
+  fontWeight('font-bold'),
+  fontSize('text-lg', 'sm:text-xl', 'md:text-2xl'),
```

Comment:
`md` should have `!`, because:
1) `sm:` is applied to `>400px` screens
2) `md:` applied to `>600px` screens
3) So both are applied on big screens, which one the browser will choose?

Reply:
No, there is no need for `important` because tailwind uses `@media` queries. You can test and see that font-size change as it should